Lib/test/test_urllib2net.py

```diff
@@ -157,23 +157,23 @@ def test_file(self):
 ##             self._test_urls(urls, self._extra_handlers()+[bauth, dauth])
 
     def test_urlwithfrag(self):
-        urlwith_frag = "http://www.pythontest.net/index.html#frag"
+        urlwith_frag = f"{support.TEST_HTTP_URL}/index.html#frag"
         with support.transient_internet(urlwith_frag):
             req = urllib.request.Request(urlwith_frag)
             res = urllib.request.urlopen(req)
             self.assertEqual(res.geturl(),
-                    "http://www.pythontest.net/index.html#frag")
+                    f"{support.TEST_HTTP_URL}/index.html#frag")
 
     def test_redirect_url_withfrag(self):
-        redirect_url_with_frag = "http://www.pythontest.net/redir/with_frag/"
+        redirect_url_with_frag = f"{support.TEST_HTTP_URL}/redir/with_frag/"
```

Comment:
Ditto. Revert such change. The URL becomes wrong if TEST_HTTP_URL becomes example.org, python.org, google.com or whatever else.